packages/fxa-auth-server/lib/payments/processing-tasks-setup.ts

```diff
@@ -76,10 +81,19 @@ export async function setupProcessingTaskObjects(processName: string) {
   const stripeHelper = new StripeHelper(log, config, statsd);
   Container.set(StripeHelper, stripeHelper);
 
+  const paypalClient = new PayPalClient(
+    config.subscriptions.paypalNvpSigCredentials
+  );
+  Container.set(PayPalClient, paypalClient);
+  const paypalHelper = new PayPalHelper({ log });
+  Container.set(PayPalHelper, paypalHelper);
+
   return {
     log,
     database,
     senders,
     stripeHelper,
+    paypalHelper,
```

Comment:
technically we could get all of these from the container, a few lines above you'll see it sets stripehelper in the container as well, but i went with this since its consistenty with how these are returned from this script and how other some cron job scripts seem to be loading them in.

Reply:
i ended up removing them from this file so the other processing tasks that consumer this dont create them and instead followed the pattern in the paypal-processor